public/js/utils/sources-tree.js

```diff
@@ -59,6 +59,11 @@ function getURL(source) {
       path: "/",
       group: url
     };
+  } else if (urlObj.protocol === "webpack:") {
```

Comment:
It's probably better to whitelist the protocols where know there is a domain/host, at least as a temporary solution. Change it to:

```
} else if (urlObj.protocol === "http" || urlObj.protocol === "https") {
  return {
    path: urlObj.pathname,
    group: urlObj.host
  }
}

return {
  path: urlObj.path,
  group: urlObj.protocol + "//"
}
```